examples/SecretKnockSensor/SecretKnockSensor.ino

```diff
@@ -182,7 +182,7 @@ void listenToSecretKnock()
 
 	int currentKnockNumber = 0;               // Position counter for the array.
 	int startTime = millis();                 // Reference for when this knock started.
-	int now = millis();
+	int now;
```

Comment:
Not related to your changes, but assigning the return value of millis() to a signed integer will only work the first 32 seconds after boot. Better use unsigned long or uint32_t.